### ISSUE TITLE
What exactly is wrong with the wiki document for windows?

### ISSUE THREAD

**Issue by TheFern2** [jeremyletang/rust-sfml#320]
I have no idea where else to discuss this, I know wikis are part of the community but there is no way to discuss this anywhere else.

If you compare my edits with newer edits there are claims about being outdated and everything below is old, and @crumblingstatue rather than fixing whatever was outdated, just made an entire new paragraph. This can lead to confusion for new rust-sfml users.

https://github.com/jeremyletang/rust-sfml/wiki/Windows/_history
![image](https://github.com/jeremyletang/rust-sfml/assets/10265682/1a033a00-1752-425b-ae0a-729567d0773d)

Nearly this entire addition just copies what was already in the MSVC section, but is worded differently.

![image](https://github.com/jeremyletang/rust-sfml/assets/10265682/637996e0-b06e-450b-8f53-97c1e047ec77)

Everything works for me in the msvc section, so exactly what is old and outdated?

**Comment by crumblingstatue** [comment]
I can't really remember what prompted me to make that edit, I think it was a user being confused from reading it.

As for why I didn't delete the old content, I guess I felt bad about completely undoing the work of other wiki contributors.

As for what's wrong with them:

- There is no need to do any copying of libs into rust toolchain folder. That's completely unnecessary.  The GNU section is especially confusing referencing C:\Program Files, which is very likely wrong.

- No need to download LLVM. This is not necessary.

- No need to download CSFML anymore. We're using a self-hosted CSFML now.

Fell free to redo the wiki page based on this information.

**Comment by TheFern2** [comment]
I can't speak for the GNU section the outdated banner can stay for that, I only work with MSVC, but if my memory serves me right and it was a while ago when I edited the wiki I was getting an error when running cargo build and the error was pointing to needing llvm (based on the error and some googling) maybe just a misconfiguration on my end before switching toolchains can't really remember 100%. But whatever isn't needed then def can be deleted, is there because that's the only way I could compile the examples at the time. I just think having two parts addressing msvc is probably not needed. I had tested with rust-sfml 0.11.2 at the time.

I will setup a brand new windows vm, and retest the setup to remove unnecessary items. I'll edit the wiki then.
I'll add a specific rust-sfml to the wiki, really the best way to handle this would be instructions in the repo so that those can be referenced within each release.
